browser-test/src/support/admin_program_migration.ts

```diff
@@ -26,14 +26,24 @@ export class AdminProgramMigration {
     await this.page.check(`text=${adminName}`)
   }
 
+  async buttonEnabled(buttonName: string) {
+    return await this.page.getByRole('button', {name: buttonName}).isEnabled()
```

Comment:
(non-blocking, suggestion) Playwright recommends asserting if an element is enabled/disabled with the `expect(...).toBeEnabled()` (or toBeDisabled) because this format can be flaky.

Reference
- https://playwright.dev/docs/api/class-locator#locator-is-enabled
- https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-enabled

Reply:
done. thanks for the tip!